Cargo.toml

```diff
@@ -141,4 +141,8 @@ clippy.manual_let_else = "warn"
 clippy.dbg_macro = "warn"
 
 [workspace.metadata.release]
-pre-release-hook = ["/bin/sh", '-c', '/bin/sh $WORKSPACE_ROOT/scripts/add-changelog-header.sh'] # Nested use of shell to expand variables.
+pre-release-hook = [
+    "/bin/sh",
+    '-c',
+    '/bin/sh $WORKSPACE_ROOT/scripts/add-changelog-header.sh',
+] # Nested use of shell to expand variables.
```

Comment:
Formatter makes those changes automatically. I'll fix all of those unnecessary changes just before the merge

Reply:
:heavy_check_mark: